dace/codegen/targets/xilinx.py

```diff
@@ -189,6 +189,41 @@ def get_generated_codeobjects(self):
 
         return [host_code_obj] + kernel_code_objs + other_objs
 
+    def _internal_preprocess(self, sdfg: dace.SDFG):
```

Comment:
I thought we didn't need preprocess anymore...

Reply:
We do need it for this particular case